modules/ROOT/pages/patterns/primer.adoc

```diff
@@ -123,10 +123,12 @@ RETURN d.name AS nextCallingPoint
 
 |===
 
+For more information, see xref:patterns/fixed-length-patterns.adoc[].
+
 [[variable-length-paths]]
 == Matching variable-length paths
 
-Variable-length paths that only traverse relationships with a specified type can be matched with xref::patterns/concepts.adoc#quantified-relationships[quantified relationships].
+Variable-length paths that only traverse relationships with a specified type can be matched with xref::patterns/variable-length-patterns.adoc#quantified-relationships[quantified relationships].
 Any variable declared in the relationship pattern will return a list of the relationships traversed.
 The following query returns the total distance traveled via all `LINK`s connecting the stations `Peckham Rye` and `Clapham Junction`:
```

Comment:
```suggestion
The following query returns the total distance traveled via all ``LINK``s connecting the stations `Peckham Rye` and `Clapham Junction`:
```

Wrap the LINK in double backticks to fix the code formatting:

<img width="844" alt="Screenshot 2024-05-15 at 15 30 33" src="https://github.com/neo4j/docs-cypher/assets/1252064/6f105c81-3367-49b4-b57a-76d858fdadae">